hack/test-extended/forcepull/run.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
```

Comment:
There is a test-extende.sh wrapper script:  each group's run.sh gets invoked here (where you can choose to run just a given group via parameter to test-extended.sh):  https://github.com/openshift/origin/blob/master/hack/test-extended.sh#L17-L33

The vagrant plugin calls the test-extended.sh wrapper script, typically from the PR getting tagged as you noted:  https://github.com/openshift/vagrant-openshift/blob/master/lib/vagrant-openshift/action/run_openshift_tests.rb#L96-L101